### DISCUSSION TITLE
상호작용 버튼이 클릭되지 않는 현상

### DISCUSSION THREAD

**Discussion by smlee2952** [naverz/zepeto-studio-kor#1024] [5. Q&A]
현재 상호작용 버튼을 구현하고 있습니다.
가까이 가면 나타나는 것은 되었으나 버튼이 클릭이 되지 않습니다.

다음 그림과 같이 Canvas에 Render Mode를 World Space로 설정 후, Event Camera를 ZepetoCamera로 설정하였으나
버튼이 클릭되지 않습니다.

혹시 이유를 아시는 분 계실까요?

![image](https://user-images.githubusercontent.com/49216499/179359692-ac458c69-751b-4a41-9865-35a6271ef589.png)
![image](https://user-images.githubusercontent.com/49216499/179359302-0a50c3a1-f10b-4f21-b803-1378a2a88860.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Screen Space로 바꾸면 버튼 클릭이 잘되는지 먼저 확인 부탁 드립니다.

**Reply by smlee2952** [reply]
Screen Space로 변경 후 버튼 클릭이 되는 것을 확인하였습니다.

![image](https://user-images.githubusercontent.com/49216499/179503839-9fa3c28e-5c7c-44b9-8dc7-70a4f6e924db.png)

**Comment by smlee2952** [comment]
자답입니다.

상호작용 버튼을 움직여보며 테스트하다가 반대편에서 클릭을 했을 때 동작하는 것을 확인하였습니다.
상호작용 버튼이 뒤집혀 있었어서 발생한 문제였습니다.

**Reply by Zepeto-TechEvangelist** [reply]
다행입니다~ 로직은 맞게 짜신것 같아서 코드 오류는 아닌지 먼저 확인하려고 말씀드렸었고 
코드 오류가 아니니 세팅 문제라고 생각했습니다.

**Reply by smlee2952** [reply]
답변해주셔서 감사합니다!
